integration

```diff
@@ -790,6 +790,7 @@
   "rob196/home-assistant-fxmarketapi",
   "robbinjanssen/home-assistant-ojmicroline-thermostat",
   "robbinjanssen/home-assistant-omnik-inverter",
+  "robbrad/UKBinCollectionData",
```

Comment:
HACS can not detect any releases, only pre-releases of this repository, so it can not be included yet.

Reply:
@ludeeus thank you. Just published non pre-release, release